cmake/ModulePackage.cmake

```diff
@@ -62,10 +62,14 @@ function(add_module lib)
   if(ENABLE_SHARED)
     add_library(${lib}_shared SHARED ${ARGN})
     set_target_properties(${lib}_shared PROPERTIES OUTPUT_NAME ${lib})
+    add_library(${lib} ALIAS ${lib}_shared)
```

Comment:
can we have namespaced aliases?
`ToxCore::${lib}`

see https://cmake.org/cmake/help/v3.11/manual/cmake-buildsystem.7.html#alias-targets

should make it easier to conditionally fetchconent/submodule toxcore in other projects

Reply:
Yes, but not in this PR.